mutacc/utils/fastq_parse.py

```diff
@@ -0,0 +1,94 @@
+import gzip
+from pathlib import Path
+from contextlib import ExitStack
+
+from Bio.SeqIO.QualityIO import FastqGeneralIterator
+
+def fastq_extract(fastq_files: list, record_ids: set, dir_path = ''):
+
+    """
+
+        Given a list of read identifiers, and one or two (for paired end) fastq files, 
+        creates new fastq files only containing the reads specified. 
+
+        Args:
+            
+            fastq_files (list): List of fastqfiles
+            record_ids (set): Set of read names
+            dir_path (string): path to directory where new fastq files are written to
+        
+        Returns:
+
+            out_paths (list): List of paths to newly created fastq files
+
+    """
+
+
+    file_names = [Path(file_name).name for file_name in fastq_files]
+    dir_path = Path(dir_path).expanduser().absolute()
```

Comment:
Would be nice with a comment on this lines behaviour (Expanduser path and abs path )